packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx

```diff
@@ -44,13 +44,13 @@ export const WriteOnlyFunctionForm = ({
     isLoading,
     writeAsync,
   } = useContractWrite({
-    addressOrName: contractAddress,
+    address: parseAddressTo0x(contractAddress),
     functionName: functionFragment.name,
-    contractInterface: [functionFragment],
+    abi: [functionFragment],
     args: keys.map(key => form[key]),
     mode: "recklesslyUnprepared",
     overrides: {
-      value: txValue,
+      value: parseTxnValue(txValue),
```

Comment:
### Please take a look at `parseTxnValue` function. 

### Reasong for adding : 
In new version of wagmi `value` is of type - `BigNumber | number | undefined`